min-turnup/phase2/Kconfig

```diff
@@ -3,14 +3,18 @@ menu "Phase 2: Node Bootstrapping"
 
 config phase2.docker_registry
 	string "docker registry"
-	default "gcr.io/google-containers"
+	default "docker.io/colemickens"
 	help
 	  The docker registry to pull cluster components from.
 
 config phase2.kubernetes_version
 	string "kubernetes version"
-	default "v1.2.4"
+	default "v1.3.0-azure"
```

Comment:
> If it were possible to use flannel (without requiring the docker bootstrap)

With Weave Net you don't have to care about second Docker daemon, cause Weave Net doesn't reconfigure the bridge. Happy to help. Azure route configuration is plausible feature request for Weave Net, I think, but that's a separate topic.